pyne/alara.py

```diff
@@ -890,5 +892,242 @@ def _find_phsrc_dc(idc, phtn_src_dc):
                 return phtn_src_dc[i]
         raise ValueError('Decay time {0} not found in phtn_src file'.format(idc)) 
 
+def _gt_normalize(neutron_spectrum):
+    """
+    Function that normalizes the neutron spectrum for ALARA fluxin file
+    """
+    tol = 1E-8
+    total = float(np.sum(neutron_spectrum))
```

Comment:
neutron_spectrum is created in step0 in gtcadis.py. It's just a list of 175 1's as [1]*175. I guess this way the function is more generic in case it'll be used for normalization of other spectrums.